### PR TITLE
`General`: Improve exercise header

### PR DESCRIPTION
**Pull request by JohannesStoehr**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

The exercise header provides important information about the exercise but should not take up too much space while doing so. This PR improves this further.

### Description

The exercise header now provides information about the achieved points in the exercise.
The left column of the header was often longer than the right column. To counter this two short rows of the left column are now in the same line when there is enough width.
The tooltips explaining the assessment types were improved.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Exercise (Modeling or Text)
- 1 Programming Exercise 

1. Log in to Artemis
2. Create the exercise
3. Participate in the exercises as student
4. Grade the modeling/text exercise
5. Check the page showing the assessment as student and check the header there as well
6. Check the header of both exercises starting in step 3

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [x] Test 1
- [ ] Test 2

### Screenshots

Bildschirmfoto 2022-09-28 um 20 42 01

### CODE REVIEW

**Review by stefanwaldhauser — APPROVED** [review]
Tested it on TS1. Great Changes! The only thing you might want to do is to add a small unit test to test stuff like the points showing or such. But imo not necesssary for these small changes

File: src/main/webapp/app/exercises/shared/exercise-headers/header-participation-page.component.ts [hunk 1]
```diff
@@ -56,6 +56,12 @@ export class HeaderParticipationPageComponent implements OnInit, OnChanges {
             this.exerciseStatusBadge = hasExerciseDueDatePassed(this.exercise, this.participation) ? 'bg-danger' : 'bg-success';
             this.exerciseCategories = this.exercise.categories || [];
             this.dueDate = getExerciseDueDate(this.exercise, this.participation);
+            if (this.participation?.results?.[0]?.rated) {
```
Comment (ge65cer) [inline src/main/webapp/app/exercises/shared/exercise-headers/header-participation-page.component.ts:59]: Q: Is the result at index 0 always the latest (rated) result?

**Review by ge65cer — APPROVED** [review]
Tested on TS1, all good!

File: src/main/webapp/app/exercises/shared/exercise-headers/header-participation-page.component.ts (hunk 1 above)
Comment (JohannesStoehr) [reply src/main/webapp/app/exercises/shared/exercise-headers/header-participation-page.component.ts:59]: Practice runs are now in their own participation

**Pull request merged**